CHANGELOG.md

```diff
@@ -1,18 +1,22 @@
 # Change Log
 
 ## [v0.1.0-beta.2](https://github.com/auth0/auth0-mcp-server/tree/v0.1.0-beta.2) (2025-04-24)
+
```

Comment:
This is coming from our `format` script. Why wasn't this failing CI, because lint doesn't happen on release? Should we tweak the settings to ignore this file?